app/src/main/java/com/asfoundation/wallet/ui/iab/RegularBuyFragment.java

```diff
@@ -0,0 +1,386 @@
+package com.asfoundation.wallet.ui.iab;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringRes;
+import android.support.constraint.Group;
+import android.util.Pair;
+import android.view.ContextThemeWrapper;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.ImageView;
+import android.widget.Spinner;
+import android.widget.TextView;
+import com.asf.wallet.R;
+import com.asfoundation.wallet.entity.TransactionBuilder;
+import com.jakewharton.rxbinding2.view.RxView;
+import com.jakewharton.rxrelay2.PublishRelay;
+import dagger.android.support.DaggerFragment;
+import io.reactivex.Observable;
+import io.reactivex.Single;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.schedulers.Schedulers;
+import io.reactivex.subjects.BehaviorSubject;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Formatter;
+import java.util.List;
+import java.util.Locale;
+import javax.inject.Inject;
+
+/**
+ * Created by franciscocalado on 19/07/2018.
+ */
+
+public class RegularBuyFragment extends DaggerFragment implements RegularBuyView {
```

Comment:
the name of this fragment doesn't help that much identifying the type of payment it does